### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by s1lvax** [ansxuman/clave#10] [Ideas]
Hi there @ansxuman ,

First of all, very nice project and very nice idea. 

I think this project has some major potential so I'd love to contribute.

Any way you could please generate a CONTRIBUTING.md with some insights ?

Maybe even a Discord server would be very helpful if you're looking for more contributors :)).

Thanks in advance !

**Comment by baptistebisson** [comment]
Hello, same for me. Would be great to also have documentation about how to build the app :+1:

**Comment by ansxuman** [comment]
Hey thanks @s1lvax @baptistebisson for highlighting this! I've added Project Setup and Contribution Guide in the `README.md`. I do have a Discord server and  I'll consider creating a dedicated channel for Clave based on the response to this project,

**Reply by baptistebisson** [reply]
Nice ! Are you going to create issues about what you want to add/fix ? Would be easier to contribute I think

**Reply by s1lvax** [reply]
If you do, kindly paste it here and I'll join :)) @ansxuman

**Reply by ansxuman** [reply]
@s1lvax @baptistebisson, I plan to add a roadmap for this project. Please give me some time, and I’ll update you on both the Discord channel and the roadmap once they’re ready.
